Easydict/NewApp/EasydictApp.swift

```diff
@@ -47,7 +42,7 @@ struct EasydictApp: App {
                 Label {
                     Text("Easydict")
                 } icon: {
-                    Image(menuBarImage)
+                    Image(menuBarIcon.rawValue)
                         .resizable()
                         .renderingMode(.template)
                         .scaledToFit()
```

Comment:
Currently, both menu bar icons can be used in debug and release mode, can we make a small modification to make it easier to distinguish between the two modes? For example, adding a tint color for debug, or something else?

Reply:
see 22adde0e